python/py-notify-python/files/missing-python27-symbols.patch

```diff
@@ -0,0 +1,11 @@
+--- src/Makefile
++++ src/Makefile
+@@ -123,7 +123,7 @@ LN_S = ln -s
+ LTLIBOBJS =
+ MAKEINFO = ${SHELL} /opt/local/var/macports/build/_opt_local_var_macports_sources_rsync.macports.org_macports_release_tarballs_ports_python_py-notify-python/py27-notify-python/work/notify-python-0.1.1/missing --run makeinfo
+ NOTIFY_PYTHON_CFLAGS = -I/opt/local/Library/Frameworks/Python.framework/Versions/2.7/include/pygtk-2.0 -I/opt/local/include/gtk-2.0 -I/opt/local/lib/gtk-2.0/include -I/opt/local/include/pango-1.0 -I/opt/local/include/gio-unix-2.0 -I/opt/local/include/cairo -I/opt/local/include/atk-1.0 -I/opt/local/include/pango-1.0 -I/opt/local/include/harfbuzz -I/opt/local/include/pango-1.0 -I/opt/local/include/fribidi -I/opt/local/include/cairo -I/opt/local/include/pixman-1 -I/opt/local/include/harfbuzz -I/opt/local/include/freetype2 -I/opt/local/include/gdk-pixbuf-2.0 -I/opt/local/include/libpng16 -I/opt/local/include/glib-2.0 -I/opt/local/lib/glib-2.0/include
+-NOTIFY_PYTHON_LIBS = -L/opt/local/lib -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -lXrender -lXinerama -lXi -lXrandr -lXcursor -lXcomposite -lXdamage -lXfixes -lX11 -lXext -latk-1.0 -lcairo -lpangoft2-1.0 -lpango-1.0 -lharfbuzz -lfontconfig -lfreetype -lnotify -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lintl
++NOTIFY_PYTHON_LIBS = -L/opt/local/lib -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -lXrender -lXinerama -lXi -lXrandr -lXcursor -lXcomposite -lXdamage -lXfixes -lX11 -lXext -latk-1.0 -lcairo -lpangoft2-1.0 -lpango-1.0 -lharfbuzz -lfontconfig -lfreetype -lnotify -lgdk_pixbuf-2.0 -lgio-2.0 -lgobject-2.0 -lglib-2.0 -lintl -lpython2.7
```

Comment:
these changes do not look correct; I guess you either have to patch the `Makefile.in` or something like that or you will need to path the generate `Makefile` as you do here, but then you need to take care of the fact that not everyone will have MacPorts installed in the `opt/local` prefix.

Reply:
hm. `NOTIFY_PYTHON_LIBS` is not defined in `Makefile.in`

Do you prefer I append in generated makefile or figure out where the variable comes from and patch there?

Reply:
However I can either append to the variable in `configure` or in the `Makefile`. what do you prefer?